Marcio/registrar revamp (#2052)
cloudflare/cloudflare-docs: products/registrar/src/content/setup-domain-transfers/index.md

---
order: 2
pcx-content-type: concept
---

import BeforeYouBegin from "../_partials/_before-you-begin.md"

# Transfer your domain to Cloudflare

Domain transfers tell your registry that a different registrar can now set those authoritative records for you. The relationship is based on trust. Registries only trust one registrar at any given time to make changes on your behalf.

> Transferring a domain to a new registrar informs the registry that they should instead trust that new registrar to modify information. The process requires some steps at both your new registrar and the one you are leaving. Each registrar handles transfers a bit differently, but in general they follow a pattern based on rules set by ICANN, the organization responsible for regulating domain registration.

Reviewer: ```suggestion
Transferring a domain to a new registrar informs the registry that they should instead trust that new registrar to modify information. The process requires some action steps at your new and previous registrar. Each registrar handles transfers a bit differently, but in general, they follow a pattern based on rules set by ICANN, the organization responsible for regulating domain registration.
```
